### ISSUE TITLE
Hungarian translation

### ISSUE THREAD

**Issue by kelna** [indywidualny/FaceSlim#226]
I translated the app to Hungarian. Here it is:
https://gist.github.com/4530645b4b913cfead176d9526ad6ee2

**Comment by indywidualny** [comment]
Thanks :)

pon., 26.12.2016, 10:47 użytkownik kelna <notifications@github.com> napisał:

**Comment by indywidualny** [comment]
I think string arrays are missing. I mean main menu and time intervals for notifications.

**Comment by kelna** [comment]
You are right, I will look into it. I used Stringlate to translate and it overlooked the string-arrays.
